### ISSUE TITLE
Remove title from the RSS feed

### ISSUE THREAD

**Issue by tio-trom** [zedeus/nitter#425]
Currently the RSS pulls the same title and description. In my RSS feed this is shown as duplicates since they are 100% the same:
```
  <title>If you need to attend a medical appointment 🏥, here’s how you can lower your risk of catching #COVID19.⁣ 

✅ Know your risk. Lower your risk.</title>

      <description><![CDATA[<p>If you need to attend a medical appointment 🏥, here’s how you can lower your risk of catching <a href="http://bird.trom.tf/search?q=%23COVID19">#COVID19</a>.⁣ 

✅ Know your risk. Lower your risk.</p>]]></description>
```
Shouldn't be a good idea to remove the title or use something else other than the tweet itself?

**Comment by Loki-L1130** [comment]
But I think that they could better short the Title with a character limit. So, It won't mess with rules on your RSS Reader. And also seems less messy too visually.

**Comment by tio-trom** [comment]
Sure, shortening the title sounds like a good idea too!

**Comment by zedeus** [comment]
Fixed by #410
